internal/workload/manager_test.go

```diff
@@ -142,6 +143,10 @@ var _ = Describe("Manager", func() {
 			err = os.Chmod(datadir, 0444)
 			Expect(err).NotTo(HaveOccurred())
 
+			if common.IsContainerized() {
+				datadir = "/sys" // set to /sys which is always readonly
```

Comment:
why creating a directory without write permissions will not work here in this case?

Reply:
because the containers runs with `CAP_DAC_OVERRIDE`